flex-squid/src/main/java/org/sonar/flex/metrics/FileMetrics.java

```diff
@@ -31,11 +33,12 @@
   private final int numberOfClasses;
   private final int numberOfFunctions;
   private final FileLinesVisitor fileLinesVisitor = new FileLinesVisitor();
+  private final String executableLines;
 
   public FileMetrics(FlexVisitorContext context) {
     AstNode rootTree = context.rootTree();
     Objects.requireNonNull(rootTree, "Cannot compute metrics without a root tree");
-    numberOfStatements = rootTree.getDescendants(
+    List<AstNode> descendants = rootTree.getDescendants(
```

Comment:
I believe that `statements` would be a better name.

Reply:
@pynicolas renamed in https://github.com/SonarSource/sonar-flex/pull/73/commits/80e609d8038bb9e8ced8130e7cdb4b6d1b3adf24.